x-pack/plugin/esql/qa/server/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/multi_node/EsqlSpecIT.java

```diff
@@ -14,7 +14,7 @@
 
 public class EsqlSpecIT extends EsqlSpecTestCase {
     @ClassRule
-    public static ElasticsearchCluster cluster = Clusters.testCluster(spec -> {});
+    public static ElasticsearchCluster cluster = Clusters.testCluster(spec -> spec.plugin("inference-service-test"));
```

Comment:
Will everything need this plugin? should it be in `testCluster`?

Reply:
I think only this one needs it - it's the only one for `multi_node` that extends from `EsqlSpecTestCase` that loads the CSV data sets.